openpype/pipeline/create/creator_plugins.py

```diff
@@ -611,6 +634,58 @@ def get_pre_create_attr_defs(self):
         """
         return self.pre_create_attr_defs
 
+    def apply_staging_dir(self, instance):
+        """Apply staging dir with persistence to instance's transient data.
+
+        Method is called on instance creation and on instance update.
+
+        Args:
+            instance (CreatedInstance): Instance for which should be staging
+                dir applied.
+
+        Returns:
+            str: Path to staging dir.
+        """
+        create_ctx = self.create_context
+        asset_name = instance.get("asset")
+        subset = instance.get("subset")
+        if not any([asset_name, subset]):
```

Comment:
This is actually faster and easier to read.
```suggestion
        if not asset_name or not subset:
```